ethcore/src/transaction_ext.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
```

Comment:
If I understand correctly, those method were not moved to common-types because of evm::Schedule dependency. Is moving evm::Schedule to common-type a possiblity (deps seems ok for that)?

Reply:
I made a mistake when checking deps, indeed it would require evm to depend on common-type.

Reply:
The thing is that I don't think it fits the transaction interface at all. IMHO it's better to have "dumb" types and implement domain-specific logic in a separate structs.
Transaction gas calculation fits more into `machine` than to transaction itself, hence the `Schedule` dependency. I didn't want to refactor the code (the PR tries to just move things around), but I think it's something worth doing in the (probably distant) future.